regression/regression-test-du-profile.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+# Runs invalid parameter handling tests
+#
+
+source /usr/local/bin/regression-suite-functions.sh
+
+# Run the command, capturing the output and RC
+factory-precaching-cli download \
+    --testmode \
+    -f "${TESTFOLDER}" \
+    --mce-version 2.2.0 \
+    -r 4.12.15 \
+    --du-profile \
+    --acm-version 2.7.0 \
```

Comment:
It seems like many of the tests are reusing many of the same versions, is it possible to de-duplicate them to a single source of truth? Or does it not matter because its creating dummy tar balls anyway?

Reply:
Updated